corehq/apps/campaign/models.py

```diff
@@ -22,13 +31,14 @@ class DashboardMap(models.Model):
         on_delete=models.CASCADE,
         related_name='maps'
     )
+    dashboard_tab = models.CharField(max_length=14, choices=DashboardTab.choices)
     display_order = models.IntegerField(default=0)
     case_type = models.CharField(max_length=255)
     geo_case_property = models.CharField(max_length=255)
 
     class Meta:
         app_label = 'campaign'
-        ordering = ['display_order']
+        ordering = ['dashboard_tab', 'display_order']
```

Comment:
Can you share why tab was added to ordering?
I assume we would filter for tab every time.

I understand, Ordering additionally by tab would add some consistency to results and probably won't hurt the query considerably but it does seem irrelevant.

Reply:
I figured there was no point not to. If I was ordering by display order, it would make no sense to mix up the dashboard tabs.

Reply:
> I figured there was no point not to

Sorry, just confirming, you meant you considered and then decided to retain dashboard_tab in ordering?